week1/homework/src/server.js

```diff
@@ -5,11 +5,41 @@ const http = require('http');
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
+
 function createServer(port) {
   let state = 10;
+  function handleRequest(response, fileType) {
```

Comment:
In the context of a server every code is in a way handling a request. So handleRequest is not a great name for a function. Can you think of a better name?

Reply:
Maybe handleResponse or sendResponse?